### PR TITLE
Add process.env variables support to services arguments.

### PR DESCRIPTION
**Pull request by barinbritva**
Resolution for https://github.com/zazoomauro/node-dependency-injection/issues/86

### CODE REVIEW

File: lib/Loader/FileLoader.js [hunk 1]
```diff
@@ -239,7 +239,11 @@ class FileLoader {
     } else if (argument.slice(0, 1) === '@') {
       return new Reference(argument.slice(1))
     } else if (argument.slice(0, 1) === '%' && argument.slice(-1) === '%') {
-      return this._container.getParameter(argument.slice(1, -1))
+      if (argument.slice(1, 4) === 'env') {
+        return process.env[argument.slice(5, -2)]
+      } else {
+        return this._container.getParameter(argument.slice(1, -1))
+      }
```
Comment (zazoomauro) [inline lib/Loader/FileLoader.js]: May we should create a private method `_getArgumentParameter(argument)` to avoid `if-returns` condensity?

**Comment by zazoomauro** [comment]
@barinbritva Thanks! Awesome job!

**Comment by zazoomauro** [comment]
I left a review. If you agree with that I can deploy a new version today.

File: lib/Loader/FileLoader.js (hunk 1 above)
Comment (barinbritva) [reply lib/Loader/FileLoader.js]: Done!

**Pull request merged**